ejb-security-programmatic-auth/src/main/java/org/jboss/as/quickstarts/ejb_security_programmatic_auth/EJBServlet.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
Please align javadoc and html content with the one you added at https://github.com/PrarthonaPaul/quickstart/blob/main/ejb-security-context-propagation/src/main/java/org/jboss/as/quickstarts/ejb_security_context_propagation/EJBServlet.java

Reply:
fixed!